Sources/VLCCloudStorageTableViewCell.m

```diff
@@ -72,6 +68,22 @@ - (void)updateOneDriveDisplayAsFolder
     _thumbnailView.image = [UIImage imageNamed:@"folder"];
 }
 
+- (void)loadThumbnail
+{
+    // The dropboxapi has no way to cancel a request and the ODThumbnail has no back reference to it's item
+    // so this might lead to wrong thumbnails if the cell is reused since we have no way of cancelling requests or check if the completion is still for the set item
+    ODDriveRequestBuilder *drive = [[ODClient loadCurrentClient] drive];
+    ODThumbnailRequest *request = [[[[drive items:_oneDriveFile.id] thumbnails:@"0"] medium] request];
+    __weak typeof(self) weakSelf = self;
+    [request getWithCompletion:^(ODThumbnail *response, NSError *error) {
+        if (error == nil && response.url) {// we don't care about errors for thumbnails
```

Comment:
Should we consider logging the error if the thumbnail fails?

Reply:
we can do debug logging I guess